.gitignore

```diff
@@ -7,5 +7,4 @@
 .idea
 .DS_Store
 .metals
-.bloop
-/jar/
\ No newline at end of file
+.bloop
```

Comment:
You no like newline 😂

Reply:
I do! Good catch! Let me add it again, it's disturbing like this :D